src/uwuw/uwuw_preprocessor.cpp

```diff
@@ -38,25 +40,26 @@ uwuw_preprocessor::~uwuw_preprocessor() {
 
 // write the new material library
 void uwuw_preprocessor::write_uwuw_materials() {
-  std::map<std::string, pyne::Material> :: iterator it;
-
-  // loop over the processed material library and write each one to the file
-  for (it = uwuw_material_library.begin() ; it != uwuw_material_library.end() ; ++it) {
-    // the current material
-    pyne::Material mat = it->second;
-    // write the material to the file
-    if (verbose) {
-      std::cout << "writing material, " << mat.metadata["name"].asString();
-      std::cout << "writing material, " << mat.metadata["fluka_name"].asString();
+  if (verbose) {
+    pyne::mat_map mat_lib_obj = uwuw_material_library.get_mat_library();
+    pyne::mat_map::iterator it;
+
+    // loop over the processed material library and write each one to the file
+    for (it = mat_lib_obj.begin(); it != mat_lib_obj.end(); ++it) {
+      // the current material
+      std::shared_ptr<pyne::Material> mat = it->second;
+      std::cout << "writing material, " << mat->metadata["name"].asString();
```

Comment:
Originally, this output occurred immediately before writing each material.  I wonder if we should reword it slightly since the writing of the HDF5 material now occurs after all this output happens?  If someone uses this for debugging, there will be a gap in logical time between these messages and the actual writing???  not sure...